examples/deployment/src/assembly/image-assembly.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+
+<!--
+
+    Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
+    Licensed under the Universal Permissive License v 1.0 as shown at
+    http://oss.oracle.com/licenses/upl.
+
+-->
+
+<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2 http://maven.apache.org/xsd/assembly-1.1.2.xsd">
+  <id>coherence-assembly</id>
+  <formats>
+    <format>dir</format>
+  </formats>
+
+  <includeBaseDirectory>false</includeBaseDirectory>
+
```

Comment:
Related to the comment on removing the dependency plugin.
Add the following here:
```
  <moduleSets>
    <moduleSet>
      <useAllReactorProjects>true</useAllReactorProjects>
      <includes>
        <include>com.oracle.coherence.kubernetes:deployment-example</include>
      </includes>
      <binaries>
        <outputDirectory>lib</outputDirectory>
        <unpack>false</unpack>
        <outputFileNameMapping>${module.artifactId}.${module.extension}</outputFileNameMapping>
        <dependencySets>
          <dependencySet>
            <outputFileNameMapping>${artifact.artifactId}.${artifact.extension}</outputFileNameMapping>
          </dependencySet>
        </dependencySets>
      </binaries>
    </moduleSet>
  </moduleSets>
```

Reply:
fixed

Reply:
fixed